crates/primitives/src/block/account.rs

```diff
@@ -41,27 +41,37 @@ impl Account {
 /// Account data from RPC `eth_getProof` response
 #[derive(Serialize, Deserialize, Debug, Clone)]
 #[serde(rename_all = "camelCase")]
-pub struct AccountFromRpc {
+pub struct AccountProofFromRpc {
```

Comment:
Alloy types can be used here?

Reply:
https://github.com/HerodotusDev/hdp/issues/95 imo it requires whole dependency changes related to `alloy`, and what i checked last header type doesn't support type cast. so excluded in this PR scope